src/renderer/tvgScene.h

```diff
@@ -77,6 +84,11 @@ struct Scene::Impl
         if (auto renderer = PP(scene)->renderer) {
             renderer->dispose(rd);
         }
+
+        if (vport) {
```

Comment:
--null check

Reply:
I need to check it somewhere, but now I've compressed it:
```
        if (vport) delete(vport->vclip);
        free(vport);
```